### PR TITLE
Adding check for standalone project.

### PR DESCRIPTION
**Pull request by pshriwise**
This PR partially addresses #60 by re-introducing #43 with the `master` branch as its target.

Things may have changed in our build system since then, but I think this check is still valid.

### CODE REVIEW

**Comment by gonuke** [comment]
@zxkjack123 may be the best to check this since he has been working on improving the stand alone capability

**Comment by pshriwise** [comment]
I think this problem is related to #61. I'll track down the source of the problem. I used the dockerfile (very helpful btw. thank you) to check the CMake and everything seemed fine. Back soon with info on the build error...

**Comment by pshriwise** [comment]
Don't merge just yet please. I think I've addressed that bug, but the way I've worked around the `const FillNode*` here is ugly and I'd like to fix that before it goes in.

**Comment by pshriwise** [comment]
Ok so I ended up pulling that last commit from this branch. I decided to go ahead and fix the lattice translation problem for all transforms, not just translations, which involved bringing in some matrix math from Eigen. As we don't want to now have two LA libraries as dependencies of this tool, I also went back and took a crack at finishing up previous work on using Eigen for GQ characterization and it went fairly well. 

In short, I *think* I have a working version of `mcnp2cad` which now relies only on Eigen for linear algebra. I'll submit it as a separate PR soon.

**Comment by gonuke** [comment]
pinging @zxkjack123 since he was also looking into Eigen...

**Comment by pshriwise** [comment]
This PR has been superseded by #68. Closing...
